hsapp/menubar.go

```diff
@@ -182,6 +184,14 @@ func (a *App) onFilePreferencesClicked() {
 	a.preferencesDialog.Show(a.config)
 }
 
+func (a *App) onExitProjectClicked() {
+	a.project = nil
```

Comment:
maybe also call `a.project.Save()` here

Reply:
I feel that there should be a dialog telling the user: "Do you want to save the project before closing? Yes/No/Cancel"
This way, if the user wanted to get rid of all the changes, that they made during the hellspawner session, they would have the option to do so.

Reply:
for yes-no dialog, you can use `dialog` package we use in a few other places:
```golang
if save := dialog.Message("Do you want to save current project?").YesNo(); save {
       a.project.Save()
}
```

Reply:
I will implement the dialog box in another PR.